### PR TITLE
Added user test inline command

### PR DESCRIPTION
**Pull request by armanio123**
Added new `@typescript-bot user test this inline` command.

### CODE REVIEW

File: GithubCommentReader/index.js [hunk 1]
```diff
@@ -150,7 +161,7 @@ async function triggerGHActionWithComment(request, event, payload, message) {
     const workflow = await cli.actions.listRepoWorkflowRuns({
         owner: "microsoft", 
         repo: "TypeScript",
-        branch: "main",
+        branch: "master",
```
Comment (weswigham) [inline GithubCommentReader/index.js:164]: You'll probably need to swap this back~

File: GithubCommentReader/index.js [hunk 2]
```diff
@@ -351,7 +377,7 @@ const commands = (/** @type {Map<RegExp, CommentAction>} */(new Map()))
         const branch = `release-${match[1]}`;
         await triggerGHActionWithComment(request, "sync-branch", {
             branch_name: branch
-        }, `sync \`${branch}\` with main`);
+        }, `sync \`${branch}\` with master`);
```
Comment (weswigham) [inline GithubCommentReader/index.js:380]: This probably also needs to be swapped back to `main`?

File: GithubCommentReader/index.js [hunk 3]
```diff
@@ -242,7 +253,7 @@ const commands = (/** @type {Map<RegExp, CommentAction>} */(new Map()))
             target_branch: pr.head.ref
         })};
     })))
-    .set(/user test this(?! slower)/, action(async request => await makeNewBuildWithComments(request, "parallelized community code test suite", 33, async p => {
+    .set(/user test this(?! slower|inline)/, action(async request => await makeNewBuildWithComments(request, "parallelized community code test suite", 33, async p => {
```
Comment (weswigham) [inline GithubCommentReader/index.js:256]:
```suggestion
    .set(/user test this(?! slower| inline)/, action(async request => await makeNewBuildWithComments(request, "parallelized community code test suite", 33, async p => {

```
The whitespace in the regex has to be in both branches of the `|`.

**Review by weswigham — CHANGES_REQUESTED** [review]

File: GithubCommentReader/index.js (hunk 1 above)
Comment (armanio123) [reply GithubCommentReader/index.js:164]: Wow, I don't remember changing this. Fixed.

File: GithubCommentReader/index.js (hunk 3 above)
Comment (armanio123) [reply GithubCommentReader/index.js:256]: You're right. Fixed.

File: GithubCommentReader/index.js (hunk 2 above)
Comment (armanio123) [reply GithubCommentReader/index.js:380]: Fixed.

**Pull request merged**